test/e2e/page-objects/pages/dialog/dialog.ts

```diff
@@ -0,0 +1,20 @@
+import { Driver } from '../../../webdriver/driver';
+import { strict as assert } from 'assert';
+
+class GenericConfirmCancelDialog {
```

Comment:
given that this is used in the add token spec, I think we could name this as AddTokenDialog, to avoid using generic classes where we might not know where to use in the future :thinking: 
This way, we could also add a couple of `checks` for the page load, like the title is there etc, so whenever the dialog is open, we could add this

`check_pageIsLoaded` like we do in other page object classes

What do you think?
What do you think?

Reply:
I did also create a page for add token, but IIRC, we have a lof of modals which just need to `confirm or cancel` / `approve or cancel`

My thinking was to maybe create a generic one that could be used for simple dialogs?

Reply:
the advantage to create more specific one is that we can check all the displayed information for the specific dialog, it will be easier to maintain. Or what we can do it to create a generic one, then other dialog can extend the generic one.Both works for me.

Reply:
sounds good, thanks !